docs/source/developer/index.rst

```diff
@@ -36,10 +36,16 @@ Then::
 
 Generate the Library
 ~~~~~~~~~~~~~~~~~~~~
+First make sure the swagger.json file is up to date. It contains the OpenAPI specification of the API for which we want to generate the client code.
```

Comment:
Is there any way to pull the `swagger.json` from a URL? Including a copy in the client repository means it can go out of sync from the server implementation.

I feel like there should be a single "source of truth" when it comes to `swagger.json` and not multiple instances of the file in multiple repositories 🤔

Reply:
Yes that is a good point we need to tackle, especially that would make this repo independant (version wise) from the specification.

Reply:
I'll keep that for later